.github/workflows/backport.yml

```diff
@@ -1,16 +1,20 @@
 name: Backport
 on:
-  pull_request:
+  pull_request_target:
     types:
       - closed
```

Comment:
What happens if, for example, some PR gets labeled as `v1.2.5`, but then gets closed without merging - will this still open a backport PR based off of it?

Reply:
The GHA will open PRs only for merged PRs.